### PR TITLE
wip: prepare changelog for 0.14.0 release

### PR DESCRIPTION
**Pull request by vanbasten17**
_Set as [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) if it's not ready to be merged_.  

[PR best practices Reference](https://blog.codeminer42.com/on-writing-a-great-pull-request-37c60ce6f31d/)

## Description
WIP changelog to be added in the next Botonic release.

### CODE REVIEW

**Comment by vanbasten17** [comment]
@ericmarcos, now that we added the webchat-entry should we do the same for webviews, node, ... in order to reduce even more the size of the generated `dist` folder?

**Comment by ericmarcos** [comment]
> @ericmarcos, now that we added the webchat-entry should we do the same for webviews, node, ... in order to reduce even more the size of the generated `dist` folder?

Yes!!

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,79 @@ All notable changes to Botonic will be documented in this file.
...
+  - Passing `mobileStyle` in `webchat/index.js`is now allowed.
+  - Added arrow buttons for better interaction with Webchat Carousels. They can be customized.
+  - Disabling timestamps in the custom Messages using the `enableTimestamps` prop is now allowed.
+  - Defined where to store the botonicState using the variable storage.
+  - The methods `openCoverComponent`, `closeCoverComponent` and `toggleCoverComponent` are available in the browser, through the Botonic object, so that the `coverComponent` can be shown/hidden manually.
+
+- [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
+
+  - Decorator for Contentful that can distribute the delivery requests amongst different spaces/environments.
+  - Added a new tool to modify Contentful spaces exported as json file with "contentful space export".
```
Comment (dpinol) [inline CHANGELOG.md:39]: "modify Contentful spaces" => "modify the locales of Contentful spaces"
Append: " The tool is able to change the name of a locale without altering the contents and to remove unwanted locales

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,79 @@ All notable changes to Botonic will be documented in this file.
...
+  - Added a tool to duplicate the value of reference fields into a new locale. Reference fields (assets or links to other contents) will typically be the same for all locales, but not always. So we initially link them all to the same target for all locales.
```
Comment (dpinol) [inline CHANGELOG.md:40]: " So we initially link them all to the same target for all locales." The tool is useful to migrate a space where fallback locales are used to another where locales have no fallback

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,79 @@ All notable changes to Botonic will be documented in this file.
...
+  - Added support for locales which include the country (eg. es_ES). Now they are preprocessed as the root locale (without the country).
+  - Added an option for Contentful plugin to be able to transform the name of the locale when sent to Contentful.
+  - Added `logCall` flag to `contentfulOptions` in order to log creation config and calls performed to CMS.
+  - Added NLP support for Turkish.
+  - Added NLP support for Italian.
+  - Added Tokenizer, Lexer & Stop words for French.
```
Comment (dpinol) [inline CHANGELOG.md:46]: Added NLP support for French

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,6 +11,79 @@ All notable changes to Botonic will be documented in this file.
...
+  - Disabled text input when `CoverComponent`is shown.
+  - Added missing `animationsEnabled` prop.
+  - Now Custom Message Types can be used together with the Reply component.
+  - Deactivated markdown in webchat when viewing messages using WhatsApp provider.
+  - Sending messages with only spaces is now avoided.
+  - Renamed theme property `hoverText` to `hoverTextColor` in button.jsx so that changing the text color of a button on hover can work.
+
+- [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
+  - Fixed bug in Contentful Schedule, where schedule was miscalculating around midnight. It was not taking the timezone offset into account to calculate the weekday.
+  - Flag so that the `ContentValidator` only validates contents which are reachable.
```
Comment (dpinol) [inline CHANGELOG.md:84]: the flag is not a bug and it's not important enough to be in release notes

File: CHANGELOG.md [hunk 5]
```diff
@@ -11,6 +11,79 @@ All notable changes to Botonic will be documented in this file.
...
+  - Decorator for Contentful that can distribute the delivery requests amongst different spaces/environments.
+  - Added a new tool to modify Contentful spaces exported as json file with "contentful space export".
+  - Added a tool to duplicate the value of reference fields into a new locale. Reference fields (assets or links to other contents) will typically be the same for all locales, but not always. So we initially link them all to the same target for all locales.
+  - Added support for locales which include the country (eg. es_ES). Now they are preprocessed as the root locale (without the country).
+  - Added an option for Contentful plugin to be able to transform the name of the locale when sent to Contentful.
+  - Added `logCall` flag to `contentfulOptions` in order to log creation config and calls performed to CMS.
+  - Added NLP support for Turkish.
+  - Added NLP support for Italian.
+  - Added Tokenizer, Lexer & Stop words for French.
+
```
Comment (dpinol) [inline CHANGELOG.md:47]: Added class ContentsValidator, which validates that all contents of a locale can be correctly delivered.

**Review by dpinol — CHANGES_REQUESTED** [review]

**Pull request merged**